src/classes/Inventory.ts

```diff
@@ -190,16 +190,14 @@ export default class Inventory {
         const s = SKU.fromString(sku);
 
         if (s.quality === 5) {
-            // generic getAmount so return total that match the generic sku type
-            return (
-                this.effects
-                    .map(e => {
-                        s.effect = e.id;
-                        return this.getAmount(SKU.fromObject(s), tradableOnly);
-                    })
-                    // add up total found; total is undefined to being with
-                    .reduce((total, currentTotal) => (total ? total + currentTotal : currentTotal))
-            );
+            const all = tradableOnly
+                ? Object.keys(this.tradable)
+                : Object.keys(this.tradable).concat(Object.keys(this.nonTradable));
+            return all
+                .filter(e => e.startsWith(sku))
+                .reduce((sum, s) => {
+                    return sum + this.getAmount(s, tradableOnly);
+                }, 0);
```

Comment:
I think this is a completely different thing.

Reply:
before, you were getting ammount of all variations of sku(with all effects)
now we filter all skus that have the same defindex and quality `<defindex>;5`
and get amount for them - so instead of getting amount for all existing effects, we get amount just for all effects in our pricelist.